src/main.ts

```diff
@@ -82,6 +82,7 @@ export async function activate(
 
     const csharpChannel = vscode.window.createOutputChannel('C#');
     const dotnetTestChannel = vscode.window.createOutputChannel('.NET Test Log');
+    const dotnetChannel = vscode.window.createOutputChannel('.NET');
```

Comment:
I admit I would not have thought the name here would be just '.NET'. '.NET NuGet restore?" Or something similar?

Reply:
I wanted to use the same channel O# did for nuget restore.  It makes sense from a 'shelling out to .net cli' perspective which is what O# uses it as.

Reply:
@dibarbet: I guess the Omnisharp name doesn't make sense to me either; since I wouldn't expect all dotnet commands to be dumped into the same window!

Reply:
Well - you'll be happy to learn that there actually only is 1 command in the dotnet channel in O# - restore!  Looks like I can rename it easily enough.

Reply:
I don't know why a new channel was created for it. My feeling is, since it is the C# extension performing the restore, why not just put it in the C# channel?

Reply:
The C# channel is incredibly verbose if you have logging turned on.  You'll get restore output interleaved with c# server output.  IMHO separate channel is a bit better